my-app/src/css/popup.css

```diff
@@ -0,0 +1,20 @@
+label{
+	margin-left: 5px;
+}
+
+#input{
+	margin-top: 12px;
+}
+
+#submit{
+	min-width: 120px;
+}
+
+#JoinButton{
```

Comment:
be consistent with naming. This one starts with uppercase, but the others all start with lowercase